index.html

```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+---
+
+<div class="home">
+
+  <img src="img/starterkit.png" alt="" />
+  <p>
+    The LHCb Starterkit is an annual workshop for Master students and early PhD students. It covers basic tools such as Bash, Git, and Python as well as LHCb specific software projects.
```

Comment:
I'd remove the annual. Presumably we put the June event in the starterkit brand as well? Otherwise we need a webpage for that too :-/